src/utils/atoms/createResourceAtom.ts

```diff
@@ -135,6 +135,13 @@ function createResourceFetcherAtom<P, T>(
               requestAction = create('error', e.message ?? e ?? true);
             }
           } finally {
+            // that's when request was canceled
+            if (version !== ctx.version) {
+              // extra dispatch allows resource subscribers to proccess cancel event and then
+              // process the response of the next event
+              if (ctx.allowCancel)
+                dispatch(create('cancel', nextParams, params));
```

Comment:
the logic is a bit complicated here and I may not be correct but it looks like the code here can be like:
`dispatch(create('cancel', ctx.lastParams, params));` 
where ctx.lastParams is the latest params of new request and params are previous request params

Reply:
`lastParams` confuses me a bit
what if here we would call them `nextParams` and `prevParams` so that we would definetly see the difference? Or `latestsParams` and `prevParams`?